components/site-navigation.vue

```diff
@@ -138,7 +154,8 @@ export default {
 
 .site-navigation__list-item--active:hover,
 .site-navigation__list-item--active:active,
-.site-navigation__list-item--active:focus {
+.site-navigation__list-item--active:focus,
```

Comment:
Is `site-navigation__list-item--active` still used?

Reply:
now it is again